javascript/javascript1/week2/Ali-Sharifi/script2.js

```diff
@@ -0,0 +1,126 @@
+//Formal fullname
+function getFullname(firstname, lastname, useFormalName = false, gender = "") {
+  firstname = firstname.trim();
+  lastname = lastname.trim();
+
+  if (!firstname && !lastname) {
+    return "Please provide a name";
+  }
```

Comment:
Good check. You can improve it a bit by changing `&&` to `||`, as getting a full name depends on having both names.
```suggestion
  if (!firstname || !lastname) {
    return "Please provide a name";
  }
```